Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "winapi"
-version = "0.3.3"
+version = "0.3.4"
```

Comment:
Why exactly is this commit changing this?

Reply:
For crates that I maintain I prefer PRs to include a version number increment, so I don’t need to make another commit before running `cargo publish`. But it sounds like that’s not your preference, I’ll revert it.

Reply:
There's also the fact that the version already is 0.3.4 which makes me somewhat confused.

Reply:
>There's also the fact that the version already is 0.3.4 which makes me somewhat confused.

Because the PR is based on a very old commit instead of latest 0.3.

https://github.com/retep998/winapi-rs/commits/3ec919bc76085922767670eb7904dcd81d736495